packages/component-library/src/ChartTitle/ChartTitle.js

```diff
@@ -29,10 +29,10 @@ const subtitleStyle = css`
 
 const ChartTitle = ({ title, subtitle }) =>
   title || subtitle ? (
-    <div>
-      {title ? <h3 className={titleStyle}>{title}</h3> : null}
-      {subtitle ? <span className={subtitleStyle}>{subtitle}</span> : null}
-    </div>
+    <figcaption>
+      {title ? <h2 className={titleStyle}>{title}</h2> : null}
+      {subtitle ? <h3 className={subtitleStyle}>{subtitle}</h3> : null}
+    </figcaption>
```

Comment:
Check if this is proper usage of `figcaption` (which should be wrapped in a `figure`)

Reply:
It wasn't quite right! Thankfully I had [@fstorr's awesome reference](https://fstorr.github.io/hack-oregon-civic-components/demo-card-visualization/) and I think I now have it sorted ... at least for this part.

Next step will be to see if I can capture the legend in the ChartContainer. I'll make an issue about that.